pubsub/pubsub.go

```diff
@@ -406,8 +408,8 @@ func (s *Subscription) Receive(ctx context.Context) (_ *Message, err error) {
 		// Ask for a number of messages that will give us the desired queue length.
 		// Unless we don't have information about process time (at the beginning), in
 		// which case just get one message.
-		nMessages := 1
-		if s.avgProcessTime > 0 {
+		nMessages := 2
```

Comment:
Some subscribers may really only be able to handle one message at a time. For example, the message might be a video than requires transcoding, and that process may take minutes and use all CPUs. If we download a second message, that message will always expire.

If you want to start at 2 for testing, I'm fine with that.

Reply:
Yea, good point. Done.